### PR TITLE
Fix for #185

### PR DESCRIPTION
**Pull request by kevinsafon**
Return -1 instead of throwing an exception so that it doesnt stop the
whole process and just ignore this stop update

### CODE REVIEW

File: onebusaway-transit-data-federation/src/main/java/org/onebusaway/transit_data_federation/impl/realtime/gtfs_realtime/GtfsRealtimeTripLibrary.java [hunk 1]
```diff
@@ -545,8 +545,8 @@ private int computeArrivalTime(StopTimeEntry stopTime,
       return stopTime.getArrivalTime() + arrival.getDelay();
     if (arrival.hasTime())
       return (int) (arrival.getTime() - serviceDate / 1000);
-    throw new IllegalStateException(
-        "expected arrival delay or time for stopTimeUpdate " + stopTimeUpdate);
+
```
Comment (barbeau) [inline onebusaway-transit-data-federation/src/main/java/org/onebusaway/transit_data_federation/impl/realtime/gtfs_realtime/GtfsRealtimeTripLibrary.java]: I'd recommend a comment here indicating that if we've gotten this far the GTFS-rt data is bad (i.e., violates the spec), and preferably log it so data isn't just disappearing and there is some indication of the bad data externally.

File: onebusaway-transit-data-federation/src/main/java/org/onebusaway/transit_data_federation/impl/realtime/gtfs_realtime/GtfsRealtimeTripLibrary.java [hunk 2]
```diff
@@ -545,8 +545,8 @@ private int computeArrivalTime(StopTimeEntry stopTime,
       return stopTime.getArrivalTime() + arrival.getDelay();
     if (arrival.hasTime())
       return (int) (arrival.getTime() - serviceDate / 1000);
-    throw new IllegalStateException(
-        "expected arrival delay or time for stopTimeUpdate " + stopTimeUpdate);
+	
+	return -1;
```
Comment (barbeau) [inline onebusaway-transit-data-federation/src/main/java/org/onebusaway/transit_data_federation/impl/realtime/gtfs_realtime/GtfsRealtimeTripLibrary.java]: It doesn't appear that I can comment on Github on unchanged lines of code, so I'll comment here instead - this same conceptual change should be made in `computeDepartureTime()` below - the same problem would occur with an empty departure time.  Also recommend log statement and comment there as well.

**Comment by barbeau** [comment]
Thanks for submitting this @kevinsafon!

**Comment by barbeau** [comment]
Also - I'm going to close and re-open this PR so our new ICLA bot can (hopefully) comment.  @kevinsafon please sign the ICLA linked there.

**Comment by barbeau** [comment]
Ok, @kevinsafon ICLA looks good, thanks!

**Comment by kevinsafon** [comment]
Ok, sounds good @barbeau.

I added the comment and logging in the 2 methods: computeDepartureTime and computeArrivalTime

**Comment by aaronbrethorst** [comment]
no action in 8 years. closing.

**Pull request closed without merging**